R/process_data.R

```diff
@@ -341,13 +341,21 @@ process_scenario_data <- function(data, start_year, end_year, sectors, technolog
 #'
 #' @return A tibble of data as indicated by function name.
 #' @noRd
-process_carbon_data <- function(data, start_year, end_year) {
-  data_processed <- data %>%
-    dplyr::filter(dplyr::between(.data$year, .env$start_year, .env$end_year)) %>%
-      dplyr::filter(.data$model == "MESSAGEix-GLOBIOM 1.0") %>%
-      dplyr::select(-c(scenario_geography))  %>%
-    # dplyr::filter(.data$model %in% .env$model_filter)
-    stop_if_empty(data_name = "Carbon Data")
+process_carbon_data <- function(data, start_year, end_year, carbon_price_model) {
+
+  data_processed <- data
+
+  if (is.null(data_processed)) {
```

Comment:
please add comment that explains when data_process is null

Reply:
done